bot_loader/game_starter.py

```diff
@@ -157,63 +153,63 @@ def play(self):
             print(f"map not in Maps:{new_line}{new_line.join(self.maps)}")
             return
 
-        enemy_text = args.player2
-        if enemy_text == "random":
-            enemy_text = random.choice(list(self.random_bots.keys()))
+        player2: str = args.player2
+        if player2 == "random":
+            player2 = random.choice(list(self.random_bots.keys()))
 
-        enemy_split: List[str] = enemy_text.split(".")
-        enemy_type = enemy_split.pop(0)
+        player2_split: List[str] = player2.split(".")
+        player2_type: str = player2_split.pop(0)
 
-        bot_split: List[str] = bot_text.split(".")
-        bot_type = bot_split.pop(0)
+        player1_split: List[str] = player1.split(".")
+        player1_type: str = player1_split.pop(0)
 
-        if bot_type not in self.definitions.player1:
+        if player1_type not in self.definitions.player1:
             keys = list(self.definitions.player1.keys())
-            print(f"Player1 type {bot_text} not found in:{new_line} {new_line.join(keys)}")
+            print(f"Player1 type {player1} not found in:{new_line} {new_line.join(keys)}")
             return
 
         enemy: Optional[AbstractPlayer]
 
-        if enemy_type not in self.definitions.player2:
+        if player2_type not in self.definitions.player2:
             # loader = BotLoader()
             # root_dir = os.path.dirname(os.path.abspath(__file__))
             # path = os.path.join("Bots")
             # path = os.path.join(root_dir, path)
             # loader.get_bots(path)
```

Comment:
Might as well remove this commented out code as the implementation is now somewhere else.